sycl/test/basic_tests/handler/handler_set_args.cpp

```diff
@@ -0,0 +1,255 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+//==--------------- handler_set_args.cpp -------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include <CL/sycl.hpp>
+#include <cassert>
+
+struct use_offset {
+  static const int no = 0;
+  static const int yes = 1;
+};
```

Comment:
Unless you're using fuzzy logic, I believe bool is a more suitable type here.

```suggestion
constexpr bool UseOffset = true;
constexpr bool NoOffset = false;
```

Reply:
ok